sbt-scalajs-cross/src/main/scala/scalajscross/ScalaJsCross.scala

```diff
@@ -0,0 +1,41 @@
+package scalajscross
+
+import sbtcross._
+
+import sbt._
+
+import org.scalajs.sbtplugin.{ScalaJSPlugin, ScalaJSCrossVersion}
+
+case object JSPlatform extends Platform {
```

Comment:
This `JSPlatform` should be in its own file named `JSPlatform.scala`.

In general, top-level traits, classes and objects should be in a file of their name.